assets/src/edit-story/components/library/libraryTabs.js

```diff
@@ -14,30 +14,53 @@
  * limitations under the License.
  */
 
+/**
+ * WordPress dependencies
+ */
+import { useRef, useCallback } from '@wordpress/element';
+
 /**
  * Internal dependencies
  */
+import { useKeyDownEffect } from '../keyboard';
 import useLibrary from './useLibrary';
-import { Tabs, Media, Text, Shapes, Links } from './tabs';
+import { Tabs, getPanes } from './panes';
+import { getTabId } from './panes/shared';
 
 function LibraryTabs() {
   const {
     state: { tab },
     actions: { setTab },
-    data: {
-      tabs: { MEDIA, TEXT, SHAPES, LINKS },
-    },
+    data: { tabs },
   } = useLibrary();
-  const tabs = [
-    [MEDIA, Media],
-    [TEXT, Text],
-    [SHAPES, Shapes],
-    [LINKS, Links],
-  ];
+  const panes = getPanes(tabs);
+  const ref = useRef();
+  const handleNavigation = useCallback(
+    (direction) => () => {
+      const currentIndex = panes.findIndex(({ id }) => id === tab);
+      const nextPane = panes[currentIndex + direction];
+      if (!nextPane) {
+        return;
+      }
+
+      setTab(nextPane.id);
+      const tabId = getTabId(nextPane.id);
+      ref.current.querySelector(`#${tabId}`).focus();
+    },
+    [tab, setTab, panes]
+  );
+  // todo: support RTL
```

Comment:
For reference #351

Reply:
Which change are you specifically requesting here?

Reply:
No change requested :-) Just an FYI that there‘s a related PR

Reply:
It's just that the github issue says _"@spacedmonkey requested changes"_ with only this comment... And this change request is blocking merging (besides the outstanding UX question), so I just wanted to check if that was a mistake or not.

<img width="763" alt="Screen Shot 2020-02-26 at 3 13 56 PM" src="https://user-images.githubusercontent.com/637548/75383595-a4c0ba00-58aa-11ea-848d-6ec325bfdedc.png">

Reply:
Dismissed the review now 🙂